src/main/java/com/fdmgroup/forex/repos/FxRateUpdateTimeRepo.java

```diff
@@ -0,0 +1,9 @@
+package com.fdmgroup.forex.repos;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.fdmgroup.forex.models.FxRateUpdateTime;
+
```

Comment:
Missing `@Repository` annotation?

Reply:
Resolved in [9f55491](https://github.com/forex-agile/backend/pull/19/commits/9f5549160860808140db8729345903f97e14de6a)